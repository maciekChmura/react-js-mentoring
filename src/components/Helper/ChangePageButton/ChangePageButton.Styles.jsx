import styled from 'styled-components';

const Button = styled.button`
  background-color: grey;
  height: 36px;
  width: 160px;
  border: 0;
  border-radius: 4px;
  color: white;
  font-size: 18px;

  :hover {
    color: salmon;
  }
`;

export default Button;
