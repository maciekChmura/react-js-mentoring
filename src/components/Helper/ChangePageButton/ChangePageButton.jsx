import React from 'react';
import { connect } from 'react-redux';
import { unSelectMovie } from '../../../redux/actions';
import Button from './ChangePageButton.Styles';

export const ChangePageButton = ({ unSelectMovie }) => (
  <Button onClick={() => unSelectMovie()} id="changePageButton">
    go back
  </Button>
);

export default connect(
  null,
  { unSelectMovie }
)(ChangePageButton);
