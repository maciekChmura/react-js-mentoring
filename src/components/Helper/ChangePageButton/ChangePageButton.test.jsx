import React from 'react';
import renderer from 'react-test-renderer';
import { ChangePageButton } from './ChangePageButton';

test('ChangePageButton renders', () => {
  const component = renderer.create(<ChangePageButton />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
