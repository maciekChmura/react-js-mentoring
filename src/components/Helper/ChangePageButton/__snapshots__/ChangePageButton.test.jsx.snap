// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChangePageButton renders 1`] = `
<button
  className="ChangePageButtonStyles__Button-ws550g-0 famkUL"
>
  change page
</button>
`;
